modules/appeals_api/app/workers/appeals_api/higher_level_review_pdf_submit_job.rb

```diff
@@ -31,7 +31,7 @@ def upload_to_central_mail(higher_level_review_id, pdf_path)
         'veteranLastName' => higher_level_review.last_name,
         'fileNumber' => higher_level_review.file_number.presence || higher_level_review.ssn,
         'zipCode' => higher_level_review.zip_code_5,
-        'source' => higher_level_review.consumer_name || 'lighthouse-hlr',
+        'source' => "Appeals-HLR-#{higher_level_review.consumer_name}",
```

Comment:
What happens if higher_level_review.consumer_name is nil? Do we just end up with "Appeals-HLR-" I'm not saying consumer name should ever be nil but just asking the question?

Reply:
it would be "Appeals-HLR-", which i think is fine.

Reply:
(fine for a situation that won't happen)

Reply:
Alright